data_sources/wikipedia/cases/extract_wp_case_data.py

```diff
@@ -35,7 +44,7 @@ def get_confirmed_and_deaths(content):
     df = pd.DataFrame.from_dict(
         data, orient="index", columns=["total_cases", "total_deaths"]
     )
-    df.index = pd.DatetimeIndex(df.index)
+    df.index = pd.DatetimeIndex(df.index, dayfirst=True)
```

Comment:
I think US was the year-first?

Reply:
Most of countries are year-first and dayfirst=True` is for Denmark.
When I put the `dayfirst=True`, if the first values are exceeding 31, pandas automatically consider it as year-first. So I think there is no problem for now, as far as the chart uses year as YYYY.